crates/nu_plugin_mime/src/guess.rs

```diff
@@ -124,15 +131,16 @@ impl Command for MimeGuess {
                     }
                 });
 
-                let ctrlc = engine_state.ctrlc.clone();
+                let ctrlc = compile_error!("can't figure out how to get ctrlc in plugin yet");
```

Comment:
This is the part in question where I need ctrlc to be able to make this streamable.